spec/system/planning_applications/drawing_sitemap_spec.rb

```diff
@@ -38,7 +38,6 @@
 
       click_button "Audit log"
       click_link "View all audits"
-      expect(page).to have_content("Red line drawing created")
```

Comment:
why remove this spec ?

Reply:
I unified the red line drawing audit actions, so now only the update action gets registered on the audit - since both create and update are the same its doesn't make sense to have two separate wordings for the same thing